lib/ext.Infobox.js

```diff
@@ -0,0 +1,32 @@
+var Util = require( './mediawiki.Util.js' ).Util,
+	coreutil = require('util'),
+	ExtensionHandler = require('./ext.core.ExtensionHandler.js').ExtensionHandler,
+	defines = require('./mediawiki.parser.defines.js'),
+	InfoboxRequest = require( './mediawiki.ApiRequest.js' ).InfoboxRequest;
+
+var Infobox = function() {};
```

Comment:
Well, it's just a small convention in Parsoid, separated var for dependencies and separated for class definition. In newer version of Parsoid.
In newer version of Parsoid it seems that there is even more vars :) - https://github.com/wikimedia/parsoid/blob/master/lib/ext.Cite.js#L8-L25